REQUIREMENTS-DEV.txt

```diff
@@ -1,5 +1,7 @@
 coverage
```

Comment:
This can be removed, I would imagine (I suppose pytest-cov would bring it in anyway, but we no longer depend on it directly).

Reply:
You're correct. Sorry for missing this. 

There is no coverage data reported because this PR is currently based off a feature branch. When it's swapped to master the coverage should be reported. It _is_ uploaded. The results for this commit are: https://codecov.io/github/spacetx/starfish/commit/6bdd68df06017276dcbff48df65ba6c210519c6f

For F811 to be skipped only in testing files, I found [a package](https://github.com/snoack/flake8-per-file-ignores/blob/master/test/setup.cfg) for doing it but I haven't been able to get it to work. 

You can also call # noqa: F811 line-by-line but I thought that could confuse contributors. If you prefer this I can make a follow-up PR with the change.